cogs/cryochamber.py

```diff
@@ -0,0 +1,108 @@
+"""Commands for preserving Discord messages in another channel."""
+import re
+
+from discord import Embed, Colour
+from discord.ext.commands import (Cog, CommandError, MessageConverter,
+                                  TextChannelConverter, group, check)
+
+
+class Cryochamber(Cog):
+    def __init__(self, _):
+        self.ongoing_preservations = set()
+
+    async def can_manage_messages(ctx):
```

Comment:
This function here doesn't take (or need) a `self` argument, so it can be a plain old function without needing to be nested inside the class.